tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py

```diff
@@ -548,3 +547,23 @@ def with_normalization(context, normalization_dockerfile_name: str) -> Container
     normalization_directory_with_build = normalization_directory.with_new_directory("build")
     normalization_directory_with_sshtunneling = normalization_directory_with_build.with_file("build/sshtunneling.sh", sshtunneling_file)
     return normalization_directory_with_sshtunneling.docker_build(normalization_dockerfile_name)
+
+
+def with_airbyte_python_connector(context: ConnectorContext, build_platform: Platform):
```

Comment:
@evantahler for consistency with Java connectors I decided to build Python with inline dagger instructions instead of reading their dockerfile. It means that we could eventually get rid of our connector Dockerfile.
If a developers needs to build a connector image locally they could just use:
`airbyte-ci connectors --name=source-faker build`